adapters/handlers/rest/handlers_backup.go

```diff
@@ -175,19 +175,29 @@ func (s *backupHandlers) restoreBackup(params backups.BackupsRestoreParams,
 ) middleware.Responder {
 	bucket := ""
 	path := ""
+	roleOption := models.RestoreConfigRolesOptionsNoRestore
+	userOption := models.RestoreConfigUsersOptionsNoRestore
```

Comment:
my current feeling is that we should restore if there is nothing present

Reply:
as discussed offline: we leave this for now

Reply:
following our previous discussion in chat, I think we agreed on "NoRestore"